### ISSUE TITLE
Ravencoin Wallet 3.0.0 can not imported secret words spanish to english

### ISSUE THREAD

**Issue by SebasGC77** [RavenProject/Ravencoin#1262]
This issue tracker is only for technical issues related to Ravencoin.

General Ravencoin questions and/or support requests and are best directed to the [Ravencoin Discord](https://discord.gg/jn6uhur)).

For reporting security issues, please direct message one of the core developers in discord.

### Describe the issue

I have the Ravencoin Wallet 3.0.0 app android mobile wallet installed, with mined RVN. 
I have the following problem: 
- When I want to import the Ravencoin Wallet 3.0.0 app android mobile wallet into another wallet, such as Metamask, Exodus or Trust, with the 12 secret recovery words in Spanish generated by RVN Wallet 3.0.0 app android mobile, it shows the error message: Invalid secret recovery phrase. 
Please let me know how change the language to english in the Ravencoin Wallet 3.0.0 app android mobile to get the secret words in english.

#### If so, please list the steps to reproduce below:
1.
2.
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Screenshots.
If the issue is related to the GUI, screenshots can be added to this issue via drag & drop.

### What version of Ravencoin are you using?
List the version number/commit ID, and if it is an official binary, self compiled or a distribution package such as PPA.

### Machine specs:
- OS:
- CPU:
- RAM:
- Disk size:
- Disk Type (HD/SDD):

### Any extra information that might be useful in the debugging process.
This is normally the contents of a `debug.log` or `config.log` file. Raw text or a link to a pastebin type site are preferred.

**Comment by Elsiehwoods** [comment]
@SebasGC77It sounds like you're having trouble importing your Ravencoin Wallet into other wallets using the 12 secret recovery words in Spanish.

**Comment by dicklarsson** [comment]
I dont think you can convert Spanish words to English words in a mnemonic phrase.
You have to use a wallet that supports spanish mnemonic phrases (12 words)

You have many options

1) Use this tool on Ravencoin.org, you enter your twelve words and it will show you your private/public keys
https://ravencoin.org/bip44/

2) Use a Ravencoin web wallet that supports every language in the BIP39 spec, including spanish
For eaxmple https://wallet.ravencoin.se/

**Comment by SebasGC77** [comment]
Dear Dick,

Thanks you for your support.

Thanks and regards,


Sebastián

Obtener Outlook para Android<https://aka.ms/AAb9ysg>

________________________________
De: Dick Henrik Larsson ***@***.***>
Enviado: miércoles, enero 1, 2025 8:28:53 a. m.
Para: RavenProject/Ravencoin ***@***.***>
Cc: SebasGC77 ***@***.***>; Author ***@***.***>
Asunto: Re: [RavenProject/Ravencoin] Ravencoin Wallet 3.0.0 can not imported secret words spanish to english (Issue #1262)


I dont think you can convert Spanish words to English words in a mnemonic phrase.
You have to use a wallet that supports spanish mnemonic phrases (12 words)

You have many options

  1.  Use this tool on Ravencoin.org, you enter your twelve words and it will show you your private/public keys
https://ravencoin.org/bip44/

  2.  Use a Ravencoin web wallet that supports every language in the BIP39 spec, including spanish
For eaxmple https://wallet.ravencoin.se/
